src/multibody/geometry.hpp

```diff
@@ -408,21 +363,16 @@ struct GeometryObject
     const GeometryModel & model_geom;
 
     ///
-    /// \brief Vector gathering the SE3 placements of the collision objects relative to the world.
+    /// \brief Vector gathering the SE3 placements of the geometry objects relative to the world.
     ///        See updateGeometryPlacements to update the placements.
     ///
-    std::vector<se3::SE3> oMg_collisions;
+    std::vector<se3::SE3> oMg_geometries;
```

Comment:
The _geometry is useless (it is already the meaning of the "g" in oMg. Keep the name simple: oMg.